Added section on hydroxychloroquine and chloroquine (#146)
greenelab/covid19-review: content/20.treatments.md

Summarize the state of the antiviral approach.

##### Hydroxychloroquine and Chloroquine

> Hydroxychloroquine (HCQ) and chloroquine (CQ) are quinolones that have been widely used as anti-malarial and autoimmune drugs in humans.

Reviewer: Hey Feng! Super minor but "used as anti-malarials and drugs against autoimmune disease" might be slightly more clear. "Autoimmune drugs" almost makes it sound like the drugs themselves are autoimmune.

Reviewer 2: Could just call it a DMARD (disease-modifying anti-rheumatic drug) for autoimmune diseases like lupus.

Author: Thanks for the comment, Diane! I think it's incredibly important to be as clear as possible and have made the change recommended by Rishi.

HCQ and CQ act through the same mechanism of action, but HCQ has a better safety profile [@doi:10.1136/ard.2008.101766].
HCQ/CQ have reported to have anti-viral effects, including in HIV-1, SARS-CoV, MERS-CoV, and SARS-CoV-2 [@doi:10.1016/S1473-3099(06)70361-9; @doi:10.1128/AAC.03011-14; @doi:10.1093/cid/ciaa237; @doi:10.1038/s41422-020-0282-0].
In vitro studies of CQ use for SARs-CoV infection suggest anti-viral activity is due to elevation of endosomal pH and terminal glycosylation of angiotensin-converting enzyme 2, the cellular entry receptor [@doi: 10.1186/1743-422X-2-69].
In vitro studies of SARS-CoV-2 infection show both HCQ and CQ to be effective, with HCQ being more potent [@doi:10.1093/cid/ciaa237].
Thus, there is much excitement about HCQ as a potential therapeutic for SARS-CoV-2 and COVID-19.
This section will discuss current efforts on assessing the potential role of HCQ in treating COVID-19.

> ###### Current Evidence

Reviewer 2: Hey Feng: really great work on this section, it provides a clear and effective critique of current hydroxychloroquine data. I think there are a few more trials floating around and I'll try to collect them here. 

This is one of them: Jun et al, Jrnl Zhejiang Univ. http://www.zjujournals.com/med/EN/10.3785/j.issn.1008-9292.2020.03.03

My takeaways: open-label RCT of 30 pts. No difference in viral clearance at 7d (87% HCQ vs 93% usual care), p>0.05. Big limitations: small study (1 center), surrogate endpoint, underpowered... interpret w/ caution

Author: I cited this particular one! It's the study where only the abstract is available in English. If you come across other clinical trials on HCQ (or otherwise - I know of at least one on Lopinavir-Ritonavir), I'd love if you posted here. I'm sure I'm missing some important ones.

This study also treated six patients with both HCQ + azithromycin and found this combination therapy to be more effective than HCQ alone.
This study showed design and analysis weaknesses that severely limit interpretability of results.
These weaknesses include: lack of randomization, lack of blinding, lack of placebo, lack of Intention-To-Treat analysis, lack of correction for sequential multiple comparisons, trial arms entirely confounded by hospital, false negatives in outcome measurements, lack of trial pre-registration, and small sample size.
Two of these issues are due to inappropriate data analysis and can therefore be corrected post-hoc by recalculating p-values (lack of Intention-To-Treat analysis and multiple comparisons.)
However, all other issues are fundamental design flaws and can not be corrected for.
Thus, conclusions cannot be generalized outside of the study.
Nevertheless, this study does provide promising preliminary data on the use of HCQ in COVID-19 treatment and supports further investigation of HCQ in large, rigorous clinical trials.

> A randomized, non-blinded, non-placebo trial of 62 COVID-19 patients at the Renmin Hospital of Wuhan University studied whether HCQ decreased time to fever break or time to cough relief when compared to standard care [@doi:10.1101/2020.03.22.20040758].

Reviewer 3: I believe the trial was blinded. Quoting the paper: "Neither the research performers nor the patients
were aware of the treatment assignments. " This is the last sentence in the section "Study design and participants"

Reviewer 3: Forgot to link the pdf: https://www.medrxiv.org/content/10.1101/2020.03.22.20040758v2.full.pdf

Author: I remember reading that and changing my mental note to "non-blinded" because without placebo, blinding is impossible. I should have made a note of the claim.

Trial arms were: standard care, standard care + HCQ. So at least physicians must know what they are giving and have potential to bias results. (Perhaps they pay more attention to HCQ patients. Or they are less aggressive in treating HCQ patients to give the drug time to work.) HCQ is an oral medication as well, so it's likely patients knew and could have improved due to placebo effect. 

Do you have ideas on how to succinctly and accurately phrase this distinction?

Reviewer 2: Could just call this "open-label"?

Reviewer 3: Oh that's a really good point... Perhaps you could note the claim, then add on the explanation you just mentioned here. I think that's a really useful bit of commentary, since people are a little confused about this trial. Derek Lowe's blog mentions it's blinded for example: https://blogs.sciencemag.org/pipeline/archives/2020/03/31/comparing-chloroquine-trials

Author: Done! Could you take a look?

Reviewer 3: Just took a look. This expanded discussion looks great!

A known weakness is small sample size.
This problem suggests the study is underpowered to detect potentially useful differences and precludes interpretation of results.
Additionally, because only the abstract could be read, other design and analysis issues could be present.
Conclusions should be interpreted very cautiously.
However, these preliminary negative results do reiterate the need for further study.

A case study treated 11 consecutive patients with HCQ + azithromycin using the same dosing regime reported by Gautret et al [@doi:10.1016/j.medmal.2020.03.006].
One patient died, two were transferred to the ICU, and one developed a prolonged QT interval leading to discontinuation of HCQ + azithromycin.
As in the Gautret et al study, virological clearance was assessed at Day 6 post-treatment by nasopharyngeal swabs.
Of the ten living patients at the time, eight of the patients remained positive for SARS-CoV-2 RNA.
Interpretation of conclusions are severely limited by lack of comparison group and small sample size.
However, these results stand in contrast to claims by Gautret et al and illustrates the need for better and further investigation.

> There is also conversation of clinical trials showing HCQ is superior to control treatment in inhibiting the exacerbation of pneumonia.

Reviewer 3: "conversation of clinical trials" sounds a little confusing to me as terminology. Is there a way this could be potentially reworded a bit?

Author: Is this better: "Primary literature, review articles, and media have reported HCQ is superior..."
Or just dropped that part and said: "A Letter to the Editor was published in BioScience Trends claiming numerous clinical trials showed HCQ is superior to control treatment in inhibiting the exacerbation of pneumonia."

I guess my goal was to explain why we would even be discussing this low-quality Letter that provides no data. To me, the main relevance is that people are citing/referring to it without checking the citations it makes!

Reviewer 3: I like the second formulation of a "A Letter to the editor..."! Definitely agree it's worth discussing if people are citing/referring to it...

Author: I made edits based on your suggestion. Could you take a look?

Reviewer 3: New version is much better!

These conversations cite a Letter to the Editor that makes this claim and refers to 15 pre-registration identifiers from the Chinese Clinical Trial Registry [@doi:10.5582/bst.2020.01047].
When these identifiers are followed back to the registry, most trials claim to still be recruiting patients and no trials have uploaded data or publications.

###### Summary

> 

Reviewer: What do you think about adding a section -- either as part of the summary, or before/after it, like "Summary and Further Avenues" or something -- talking about what future research regarding hydroxychloroquine and its derivatives could/should explore? (Unless there is a section elsewhere on this?)

I found this medicinal chemistry paper on hydroxychloroquine alternatives (hydroxyferroquine derivatives, which are organometallic agents) that showed selectivity against SARS-CoV back in 2006. Links here:
https://www.ncbi.nlm.nih.gov/pubmed/16640347
https://pubs.acs.org/doi/full/10.1021/jm0601856#

A section on further research on these antivirals could summarize:

What information we still need to know about hydroxychloroquine and its effectiveness / what current data are lacking (which you do here already)
Alternatives to hydroxychloroquine as antivirals that might be worth exploring (^ paper above)
Maybe... some notes on how antivirals are developed and where we could go from here? There might be another section on this part, not sure!
Also found this paper on antiviral development, which may or may not be worth citing (but putting it here to remind myself and anyone else interested, to think about):
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4765712/
https://www.elsevier.es/es-revista-medicina-universitaria-304-articulo-history-progress-antiviral-drugs-from-S166557961500037X
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2972676/pdf/main.pdf

Author: I think that could be interesting and useful, but I don't think I know enough about the overall vision for the paper to comment. @rando2, perhaps you could weigh in?

Reviewer: Makes sense. If we'd rather not add a whole section, a line or two at either 150 or 139 might be interesting just to note HCQ alternatives. For example, "Given the great focus in investigating HCQ as a potential antiviral treatment for SARS-CoV-2, it may be of interest to researchers to explore related alternatives. For example, hydroxyferroquine derivatives of HCQ have been described as a class of bioorganometallic compounds that exert antiviral effects with some selectivity for SARS-CoV."

Reviewer 4: Sorry I missed this, @hufengling and @dianerafi ! This sounds completely in line with the vision for the paper. In terms of sections, I would say either here or we could potentially add a section at the end of Therapeutics where we have people suggest different avenues that haven't been as widely explored.
I think what I'll do for now is pull this out into a separate issue, and that way we can make sure it gets integrated and loop in the people who wrote the original outline for this section to see if they have any thoughts too!

Reviewer: Thanks @rando2, that sounds great! I'll look over at the new issue now.

Reviewer 5: I think it would be great for the summary to include the study design of current trials in the summary. UPenn itself has the PATCH (Prevention and Treatment of COVID-10 with Hydroxychloroquine) trial starting soon and then there is another multi-institutional trial for in-patient population.

In vitro evidence shows HCQ may be an effective therapeutic against SARS-CoV-2 and COVID-19. 
Multiple clinical studies have already been carried out to assess this possibility.
All current studies are low-quality and have small sample sizes.
Thus, interpretation is severely limited and must be done cautiously.

> Additionally, disagreements between studies demonstrate there is controversy on the effectiveness of HCQ, as well as HCQ + azithromycin combination therapy.

Reviewer 2: Azithromycin probably merits a brief discussion here. Typically thought of as a macrolide antibiotic - so why would it have any efficacy in viral infections? As far as I can tell: Raoult (the dude who wrote that first HCQ paper) used this study as rationale: https://www.pnas.org/content/113/50/14408?ijkey=27c60509502454a97cea735745c9bb158268fffd&keytype2=tf_ipsecsha 

Shows that z-pak may be effective for ZIKV infection. I'll keep looking... there are a few papers that suggest z-pak may augment virus-induced interferon expression but they aren't super compelling

Author: Would you be able to draft some sentences on that? I don't think I have the expertise to critically read/synthesize basic immunology!

Reviewer 2: Yep of course. I remember reading the graph and saw the massive combination effect from z-pak and thinking how does that even work... then going on a frantic pubmed search to figure out why. I still think it's pretty sketchy science but there is at least *some* evidence that z-pak may have antiviral effects

Author: Hmm... Well I just absent-mindedly accepted it as fact. The beauty of collaboration!

Reviewer 2: Putting these links here as a mental bookmark to come back and write this:
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4262884/
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5918160/
https://www.ncbi.nlm.nih.gov/pubmed/28099856
https://www.ncbi.nlm.nih.gov/pubmed/27911847

Reviewer 6: ```suggestion
Additionally, disagreements between studies demonstrate there is controversy on the effectiveness of HCQ, as well as HCQ + azithromycin combination therapy.
<!-- 
rishirajgoel to add text on Azithromycin
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4262884/
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5918160/
https://www.ncbi.nlm.nih.gov/pubmed/28099856
https://www.ncbi.nlm.nih.gov/pubmed/27911847
-->
```

Great discussion.  We could use HTML comments to note these references in the text here.  That way @hufengling could finish this pull request and then @rishirajgoel could make a new one with the additional edits.  These lines should be skipped over in the rendered manuscript.

Reviewer: What do you think about adding a section -- either as part of the summary, or before/after it, like "Summary and Further Avenues" or something -- talking about what future research regarding hydroxychloroquine and its derivatives could/should explore? (Unless there is a section elsewhere on this?)

I found this medicinal chemistry paper on hydroxychloroquine alternatives (hydroxyferroquine derivatives, which are organometallic agents) that showed selectivity against SARS-CoV back in 2006. Links here:
https://www.ncbi.nlm.nih.gov/pubmed/16640347
https://pubs.acs.org/doi/full/10.1021/jm0601856#

A section on further research on these antivirals could summarize:
- What information we still need to know about hydroxychloroquine and its effectiveness / what current data are lacking (which you do here already)
- Alternatives to hydroxychloroquine as antivirals that might be worth exploring (^ paper above)
- Maybe... some notes on how antivirals are developed and where we could go from here? There might be another section on this part, not sure!

Also found this paper on antiviral development, which may or may not be worth citing (but putting it here to remind myself and anyone else interested, to think about):
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4765712/
https://www.elsevier.es/es-revista-medicina-universitaria-304-articulo-history-progress-antiviral-drugs-from-S166557961500037X
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2972676/pdf/main.pdf

Summarize the state of the antiviral approach.

##### Hydroxychloroquine and Chloroquine

Hydroxychloroquine (HCQ) and chloroquine (CQ) are quinolones that have been widely used as anti-malarial and disease-modifying anti-rheumatic drugs in humans.
HCQ and CQ act through the same mechanism of action, but HCQ has a better safety profile [@doi:10.1136/ard.2008.101766].

> HCQ/CQ have been reported to have anti-viral effects, including in HIV-1, SARS-CoV, MERS-CoV, and SARS-CoV-2 [@doi:10.1016/S1473-3099(06)70361-9; @doi:10.1128/AAC.03011-14; @doi:10.1093/cid/ciaa237; @doi:10.1038/s41422-020-0282-0].

Reviewer 4: ```suggestion
HCQ/CQ have been reported to have anti-viral effects, including in HIV-1, SARS-CoV, MERS-CoV, and SARS-CoV-2 [@doi:10/dh8phb; @doi:10.1128/AAC.03011-14; @doi:10.1093/cid/ciaa237; @doi:10.1038/s41422-020-0282-0].
```
DOIs with parentheses are a little tricky -- the reference manager in Manubot (that generates the manuscript) doesn't like them! I plugged this one into shortdoi.org to get a Manubot-friendly version

> In vitro studies of CQ use for SARs-CoV infection suggest anti-viral activity is due to inhibition of viral entry through elevation of endosomal pH and terminal glycosylation of angiotensin-converting enzyme 2, the cellular entry receptor [@doi: 10.1186/1743-422X-2-69].

Reviewer 4: ```suggestion
In vitro studies of CQ use for SARs-CoV infection suggest anti-viral activity is due to inhibition of viral entry through elevation of endosomal pH and terminal glycosylation of angiotensin-converting enzyme 2, the cellular entry receptor [@doi:10.1186/1743-422X-2-69].
```
Manubot doesn't like spaces between @doi and the doi itself

However, all other issues are fundamental design flaws and can not be corrected for.
Thus, conclusions cannot be generalized outside of the study.
Nevertheless, this study does provide promising preliminary data on the use of HCQ in COVID-19 treatment and supports further investigation of HCQ in large, rigorous clinical trials.

A randomized, non-placebo trial of 62 COVID-19 patients at the Renmin Hospital of Wuhan University studied whether HCQ decreased time to fever break or time to cough relief when compared to standard care [@doi:10.1101/2020.03.22.20040758].
This trial found HCQ decreased both time to fever break and time to cough relief.
However, this study also showed design and analysis weaknesses that disallow general interpretation of results.
These weaknesses include: lack of placebo, inappropriate choice of outcomes, lack of correction for multiple primary outcomes, lack of sufficient detail to understand analysis, drastic disparities between pre-registration and published protocol, and small sample size.
It is important to note here that the authors claim "neither the research performers nor the patients were aware of the treatment assignments."

> However, this seems impossible in a non-placebo trial - at the very least, providers would know whether they were giving HCQ or nothing, and this knowledge could lead to systematic differences in how care is given.

Reviewer 4: ```suggestion
However, this seems impossible in a non-placebo trial - at the very least, providers would know whether they were administering a medication or nothing, and this knowledge can lead to systematic differences in how care is given.
```
I think this might be what you meant -- either they gave a medication or they didn't, regardless of whether they knew it was HCQ!

Correction for multiple primary outcomes can be adjusted post-hoc by recalculating p-values, but all other issues are design and statistical problems that cannot be corrected for.
Additionally, the issue of drastic disparities between pre-registration and published protocol may be suggestive of p-hacking.
Again, conclusions cannot de generalized outside of the study, but the results support further investigation.

> A randomized trial from the Shanghai Public Health Clinical Center of 30 COVID-19 patients studied whether HCQ increased rates of virological clearance by respiratory pharyngeal swab on Day 7 post-treatment compared to standard care [@doi:10.3785/j.issn.1008-9292.2020.03.03].

Reviewer 4: ```suggestion
A randomized trial from the Shanghai Public Health Clinical Center of 30 COVID-19 patients studied whether HCQ increased rates of virological clearance by respiratory pharyngeal swab on Day 7 post-treatment compared to standard care [@doi:10/drbx].
```
Manubot didn't like this one, not sure why -- let's try the short DOI and see if it helps?

Reviewer 6: I think this error is more complicated.  I tried the original and short DOI with Manubot locally and got
```
Error in get_doi_csl_item_crosscite for 10.3785/j.issn.1008-9292.2020.03.03 due to a ConnectionError:
HTTPConnectionPool(host='data.chinadoi.cn', port=80): Max retries exceeded with url: /10.3785%2Fj.issn.1008-9292.2020.03.03 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001F5D0A91320>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',))
```

Let's leave the original DOI.  I created an issue to create a manual reference for this article #148.

### Broad-Spectrum Pharmaceuticals

##### Hydroxychloroquine and Chloroquine

Hydroxychloroquine (HCQ) and chloroquine (CQ) are quinolones that have been widely used as anti-malarial and disease-modifying anti-rheumatic drugs in humans.

> HCQ and CQ act through the same mechanism of action, but HCQ has a better safety profile [@doi:10.1136/ard.2008.101766].

Reviewer 7: 'but HCQ has a better safety profile' change to 'but HCQ is regarded to be safer. '

Reviewer 4: @nafisajadavji I just realized I can use my admin powers to convert your comments to suggestions, which should make it easier for @hufengling to incorporate them. [Here's a tutorial on how to do it](https://github.com/greenelab/covid19-review/blob/master/CONTRIBUTING.md#the-how-to-of-review)

Reviewer 7: Great, thanks!

HCQ/CQ have been reported to have anti-viral effects, including in HIV-1, SARS-CoV, MERS-CoV, and SARS-CoV-2 [@doi:10/dh8phb; @doi:10.1128/AAC.03011-14; @doi:10.1093/cid/ciaa237; @doi:10.1038/s41422-020-0282-0].

> In vitro studies of CQ use for SARs-CoV infection suggest anti-viral activity is due to inhibition of viral entry through elevation of endosomal pH and terminal glycosylation of angiotensin-converting enzyme 2, the cellular entry receptor [@doi:10.1186/1743-422X-2-69].

Reviewer 7: _In vitro_ should be _italicized_.

In vitro studies of SARS-CoV-2 infection show both HCQ and CQ to be effective in inhibiting viral replication, with HCQ being more potent [@doi:10.1093/cid/ciaa237].
Thus, there is excitement about HCQ as a potential therapeutic for SARS-CoV-2 and COVID-19.

> This section will discuss current efforts on assessing the potential role of HCQ in treating COVID-19.

Reviewer 7: change role to impact?

###### Current Evidence

Gautret et al conducted a non-randomized, non-blinded, non-placebo clinical trial on 42 hospitalized patients comparing HCQ to standard care [@doi:10.1016/j.ijantimicag.2020.105949].
This trial found patients who received HCQ showed higher rates of virological clearance by nasopharyngeal swab on Days 3-6 when compared to standard care.

> This study also treated six patients with both HCQ + azithromycin and found this combination therapy to be more effective than HCQ alone.

Reviewer 7: Should we remove this study since it is being retracted? Or add a point that it has been retracted?

Reviewer 7: It was posted to medRxiv, as a pre-print
https://www.medrxiv.org/content/10.1101/2020.03.16.20037135v1
Maybe we can use that as a citation?

Author: I'll add a point detailing the Retraction Watch post. (I don't think it's been officially retracted yet - just official comments.)

Reviewer 6: @hufengling if you like the phrasing @rando2 suggested below (https://github.com/greenelab/covid19-review/pull/146#discussion_r404739396) you could commit those changes.  I like her phrasing there.

Author: I just saw and committed those!

> This study showed design and analysis weaknesses that severely limit interpretability of results.

Reviewer 7: Change sentence to, "This study showed design and analysis weaknesses that severely limits the interpretability of results."

> These weaknesses include: lack of randomization, lack of blinding, lack of placebo, lack of Intention-To-Treat analysis, lack of correction for sequential multiple comparisons, trial arms entirely confounded by hospital, false negatives in outcome measurements, lack of trial pre-registration, and small sample size.

Reviewer 7: The weaknesses include, lack of randomization, blinding,  placebo, intention-to-treat analysis, correction for sequential multiple comparisons, and trial pre-registration. Additionally, trial arms entirely confounded by hospital, false negatives in outcome measurements, and small sample size.

> Two of these issues are due to inappropriate data analysis and can therefore be corrected post-hoc by recalculating p-values (lack of Intention-To-Treat analysis and multiple comparisons.)

Reviewer 7: Change issues to weakness to remain consistent.

> However, all other issues are fundamental design flaws and can not be corrected for.

Reviewer 7: Change issues to weaknesses.
Revised sentence: However, all other weaknesses are fundamental experimental design flaws and cannot be corrected for.

> Thus, conclusions cannot be generalized outside of the study.

Reviewer 4: ```suggestion
Thus, conclusions cannot be generalized outside of the study.
Additionally, the International Society of Antimicrobial Chemotherapy, the scientific organization that publishes _International Journal of Antimicrobial Agents_ where the article appeared, has announced that the article does not meet its expected standard for publications [@url:https://www.isac.world/news-and-publications/official-isac-statement], although it has not been officially retracted.
```

Reviewer 7: Yes, this looks good!

> Nevertheless, this study does provide promising preliminary data on the use of HCQ in COVID-19 treatment and supports further investigation of HCQ in large, rigorous clinical trials.

Reviewer 4: ```suggestion
Because of the preliminary data presented in this study, the use of HCQ in COVID-19 treatment has subsequently been explored by other researchers.
```

> A randomized, non-placebo trial of 62 COVID-19 patients at the Renmin Hospital of Wuhan University studied whether HCQ decreased time to fever break or time to cough relief when compared to standard care [@doi:10.1101/2020.03.22.20040758].

Reviewer 7: There is a difference in how males and females are affected by COVID-19, can we add details about the number of males and females in this study?

> This trial found HCQ decreased both time to fever break and time to cough relief.

Reviewer 7: Was this finding in all patients? If so, maybe add more details. The sentence seems too absolute.

> However, this study also showed design and analysis weaknesses that disallow general interpretation of results.

Reviewer 7: However, this study also showed design and analysis weaknesses that disallow general interpretation of results.
Change to 
However, this study also had flaws in experimental design and analysis, that prevent the generalization of the results.

> These weaknesses include: lack of placebo, inappropriate choice of outcomes, lack of correction for multiple primary outcomes, lack of sufficient detail to understand analysis, drastic disparities between pre-registration and published protocol, and small sample size.

Reviewer 7: Describe the inappropriate choice of outcomes.

Author: Done!
